app/views/application/_singleSymbolItems.html.haml

```diff
@@ -8,6 +8,10 @@
           - path = {controller: "admin/#{myclass.name.downcase}s", action: "edit", id: item.id}
         - else
           - path = {controller: "#{myclass.name.downcase}s", action: "show", id: item.id}
-        = button_to "#{item.name}", path, {method: :get, class: 'prod-link'}
+        -if "#{myclass.name.downcase}" == "tag"
```

Comment:
The changes to this file are unrelated to the pull request right? They just ensure that tags aren't clickable?

Reply:
They are slightly unrelated, but it is a cleanup of the forms, and thus I added it in. It just makes sure we don't open the tags edit form page because it does not exist.